wp-content/themes/darklight-site/sass/_layout/_global.scss

```diff
@@ -40,13 +33,19 @@ li {
   display:inline;
 }
 
-.header__link-item {
+.page-header {
+  text-align: center;
+  padding-top: 10px;
+  line-height: 1.8;
+
+&__link-item {
```

Comment:
because these are inside the .page-header, Id keep them indented (so inline with the line-height property